tests/wpt/meta/shadow-dom/slotchange-event.html.ini

```diff
@@ -35,18 +35,6 @@
   [slotchange event must not fire on a slot element inside a closed shadow root  not in a document when another slot's assigned nodes change]
     expected: FAIL
 
-  [slotchange event must fire on a slot element when a shadow host has a slottable and the slot was inserted and must not fire when the shadow host was mutated after the slot was removed inside an open shadow root  in a document]
```

Comment:
Do we know why so many tests in this file are still failing?

Reply:
There are some bugs in the way we assign slots that I don't fully understand yet. I looked at a few of the tests and in those cases we were firing the event correctly, but the assigned nodes were different than what was expected.

Reply:
Oh no, we're missing https://developer.mozilla.org/en-US/docs/Web/API/Element/slot... That's silly.